### DISCUSSION TITLE
Is there any inference code similar to cogvlm-chat-hf given on 🤗 face for CogAgent?

### DISCUSSION THREAD

**Discussion by OnlinePage** [THUDM/CogVLM#199] [Q&A] [answered]
Hi , first of all you guys are super awesome🙂!

I have been playing around with CogVLM models(chat and grounding genralist)

recently you guys introduced CogAgent 🤩, can you guys share its simple plain inference code as you guys have shared at [here on hugging face](https://huggingface.co/THUDM/cogvlm-grounding-generalist-hf#%E5%BF%AB%E9%80%9F%E5%BC%80%E5%A7%8B%EF%BC%88qiuckstart%EF%BC%89) for other models.

i want to deploy and run it 🙂

**Comment by zRzRzRzRzRzRzR** [comment]
We have and you can try  [Code](https://github.com/zRzRzRzRzRzRzR/CogVLM/tree/main/composite_demo) in your computer. We will release public web demo for taste next week

**Comment by wenyihong** [comment, marked as answer]
Sure! you can try [this](https://huggingface.co/THUDM/cogagent-chat-hf#quick-start) in our huggingface repo, which is the same as [this](https://github.com/THUDM/CogVLM?tab=readme-ov-file#situation-22-cli-huggingface-version) in our github repo.
